File: src/daemon/watcher/lib/xml_parser/entities/wine.ex
defmodule XmlParser.Entities.Wine do
  @type t :: %__MODULE__{
          id: String.t(),
          price: String.t(),
          designation: String.t(),
          region: String.t(),
          variaty: String.t(),
          winery: String.t(),
          title: String.t()
        }

  defstruct [:id, :price, :designation, :region, :variaty, :winery, :title]
end

File: src/daemon/migrator/examples/pkg/entities/wine.go
package entities

type Wine struct {
	Price       float32 `json:"price"`
	Designation string  `json:"designation"`
	Varaiety    string  `json:"variety"`
	Winery      string  `json:"winery"`
	Title       string  `json:"title"`
	RegionName  string  `json:"region"`
}
